packages/writing-style/package.json

```diff
@@ -20,8 +20,8 @@
   },
   "scripts": {
     "postinstall": "node scripts/download-vale.js",
-    "download:google": "node scripts/download-google-styles.js",
-    "download:writegood": "node scripts/download-writegood-styles.js"
+    "download:google": "node scripts/download-google-styles.js && prettier --write --parser yaml '**/*.{yaml,yml}'",
+    "download:writegood": "node scripts/download-writegood-styles.js && prettier --write --parser yaml '**/*.{yaml,yml}'"
```

Comment:
Hmm running prettier here is not necessary. The lint-staged hook should do that.

Reply:
It turned out to be necessary to better be able to see what has really changed in the external styles so we can adjust our own overrides respectively before committing the whole update

Reply:
Sorry, I'm not sure I fully follow the point here 🤔

Reply:
The script is pulling a large number of YAML files from an external repository to persist them as a copy in our repo (this is intended). Each line / object in them represents a certain writing style rule and some of them we override or modify in our own ruleset to adjust to our domain needs. 

When I update the external "base" rules I need a way to see what has really changed beyond formatting differences to be able to see which of our overrides I have to adjust / rename / add etc.  .   If I don't reformat the files It's all formatting noise and I don't see the actual differences.  If I reformat them I see the actual differences.    And I want to see that before committing because I don't want to commit an overall broken config.

Reply:
Thanks, I understand now. I'm fine to keep it then.

We can also avoid running prettier at all for those files, since they are downloaded anyway.